golos.publication/golos.publication.cpp

```diff
@@ -787,4 +791,35 @@ void publication::reblog(name rebloger, name author, std::string permlink) {
             "You can't reblog, because this message doesn't exist.");
 }
 
+int64_t publication::pay_delegators(uint64_t message_id, int64_t claim, name voter, 
+        eosio::symbol tokensymbol, std::vector<structures::delegate_voter> delegate_list) {
+    int64_t dlg_payout_sum = 0;
+    for (auto delegate_obj : delegate_list) {
+        if (delegate_obj.payout_strategy == config::to_delegated_vesting) {
```

Comment:
What about another strategy (pay to delegator)? Where will the reward be paid in this case?